src/WormholeFees.sol

```diff
@@ -20,6 +20,15 @@ import "./WormholeGUID.sol";
 
 // Calculate fees for a given Wormhole GUID
 interface WormholeFees {
+    /**
+    * @dev Return fee for particular wormhole. It should return 0 for wormholes that are being slow withdrawn. 
+    * note: We define slow withdrawal as wormhole older than x. x has to be enough to finalize flush (not wormhole).
+    * @param wormholeGUID - Struct which contains the whole wormhole data
+    * @param line - debt ceiling
+    * @param debt - current debt
+    * @param pending - was wormhole already registered
```

Comment:
"was wormhole already registered" -> that would be "blessed"
"pending" is the amount left to withdrawn